frontend/src/entity/Change.tsx

```diff
@@ -1,6 +1,17 @@
+import { IShortGraphQLChange } from '../service/shortGraphQL/schema';
+
 export interface Change {
   id: string;
   title: string;
   summaryMarkdown?: string;
   releasedAt: Date;
 }
+
+export function parseChange(gqlChange: IShortGraphQLChange): Change {
+  return {
+    id: gqlChange.id,
+    title: gqlChange.title,
+    summaryMarkdown: gqlChange.summaryMarkdown,
+    releasedAt: new Date(gqlChange.releasedAt)
+  };
+}
```

Comment:
I wonder whether they can move this to GraphQL package. Entities are core business logic and shouldn't depends on concrete technologies.

Reply:
Entities are the most stable and used everywhere.